scripts/add_alldata_cohort.py

```diff
@@ -612,7 +612,6 @@ def insert_barcodes_mysql(conn, superuser_id, cohort_name, sample_barcodes):
 
 
 def create_bq_cohort(project_id, dataset_id, table_id, cohort_id, sample_barcodes):
-    service = authorize_credentials_with_Google()
```

Comment:
Why is this being removed? The authorization still needs to go through. If the issue is 'service' isn't used, you just remove the variable assignment, not the call.